src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -106,6 +107,12 @@ function MoneyRequestSelectorPage(props) {
                                         component={ReceiptSelector}
                                         initialParams={{reportID, iouType}}
                                     />
+                                    {canUseDistanceRequests && (
```

Comment:
I am fine with this, but just FYI the way this is written, the user will need to have both the scan receipts and distance request beta.

Reply:
Ah yeah right. I think that's ok for now.

Reply:
Meh, might as well fix it.